routes/ft.js

```diff
@@ -40,16 +40,53 @@ router.get('/', function(req, res){
 
 			Promise.all(isRecorded)
 				.then(vals => {
+					// do a SAPI search for all artuicles tagged with the audio-articles topic
+					return getTopicArticles()
+						.then(searchResult => {
+							return{
+								vals : vals,
+								searchResult : searchResult
+							};
+						})
+					;
+				})
+				.then(valsAndSearch => {
+					// loop through search results to find all article UUIDs
+					let searchResult = valsAndSearch['searchResult'];
```

Comment:
Why have you accessed the value with square bracket notation instead of dot notation? It's not an issue, just curious.